app/views/jst/developer_key.handlebars

```diff
@@ -42,9 +42,19 @@
     </td>
     <td>
         <div>
+          {{#t "access_token_count"}}Access Token Count: {{access_token_count}}{{/t}}
```

Comment:
I get keeping the scope small; I briefly considered re-writing the entire thing in React and ditching the handlebars/coffeescript, but I figured I'd start small ;) On the other hand, it's a pretty minor change, all considered.